src/import/ImportProgressListener.h

```diff
@@ -0,0 +1,41 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  ImportProgressListener.h
+
+  V. Sverchinsky
+
+*//****************************************************************/
```

Comment:
As a matter of form I'd prefer that there is also a .cpp including this so we are sure the header has all its compilation dependencies, and the destructor should have a non-inline definition there.